renovate.json

```diff
@@ -29,7 +29,8 @@
         "*"
       ],
       "automerge": true,
-      "schedule": ["* 0-3 * * *"]
+      "semanticCommits": "enabled",
+      "semanticCommitType": "chore"
```

Comment:
This is necessary to properly update the changelog generated by release drafter

Reply:
shouldn't the commit type be `build(deps)` ?

Reply:
I do not know. I'm fine with `build(deps)` as well. I just do not want to use `fix(...)`.

Reply:
Yeah, I think may want to use `build(deps)`. https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type